frontend/src/app/network-members-resolver.service.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { TestBed } from '@angular/core/testing';
+
+import { NetworkMembersResolverService } from './network-members-resolver.service';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { NetworkMemberService } from './network-member.service';
+import { Observable, of } from 'rxjs';
+import { NetworkMemberModel } from './models/network-member.model';
+
+describe('NetworkMembersResolverService', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule]
+    });
+  });
+
+  it('should resolve network members', () => {
+    const resolver: NetworkMembersResolverService = TestBed.get(NetworkMembersResolverService);
+    const service: NetworkMemberService = TestBed.get(NetworkMemberService);
```

Comment:
Unfortunately, no. I *think* it used to return a value of the same type as the first argument, but it now returns any (I guess to deal with tokens that are not types). So I could leave out the type, but then I'd have any, so no code completion, etc.

Reply:
Damn, I did not think about that. That means I have to add them in our ebook and training too...

Reply:
After digging into Angular source code history, I'm starting to think that I hallucinated the fact that `TestBed.get(T)` ever returned `T`...

Reply:
That said, it would be a nice amendment to the API, because having to repeat the type every time is cumbersome. Something like
```
get<T>(typeToken: { new(...args: any[]): T }): T {
  return TestBed.get(typeToken);
}
```
or, if they don't want an overload, 
```
getByType<T>(type: { new(...args: any[]): T }): T {
  return TestBed.get(typeToken);
}
```

We could of course define such a global function ourselves, but it would feel more natural if it was in TestBed.

Reply:
I've been trying to come up with a PR since my comment :) I was trying to "fix" the signature (because `Injector.get(T)` *does* return `T`) but it doesn't work. I'll try an overload.